modules/ROOT/pages/class-loader-library-config.adoc

```diff
@@ -133,6 +91,54 @@ You can also configure the settings for the global library explicitly by specify
 </library>
 ----
 
+[#3rd-party]
+== Access third-party APIs from an application
+
+By default, applications do not have access to the third-party APIs that are available in Open Liberty. To enable this access, you can configure class loading for an application in your `server.xml` file.
+
+The `apiTypeVisibility` attribute of the `classloader` element specifies the types of API packages that a class loader supports. The default value for the `apiTypeVisibility` attribute includes the following API types:
+
+- `spec`: public specification APIs that are available for both compile and run time
+- `ibm-api`: APIs that are available from Open Liberty by default
+- `api`: public APIs that are available for both compile and run time
+- `stable`: stable third-party specification APIs that are available by default for both compile and run time
+
+If you want to enable access to non-default third-party APIs that are available for Open Liberty, you can specify the `third-party` API type for the `apiTypeVisibility` attribute. When you enable third-party API access, the APIs that are available to your application are still controlled by the Open Liberty features that are enabled in your `server.xml` file.
```

Comment:
Acrolinx Suggestion - non-default -> non default

Reply:
non is not a word